lang/cs/Org.Apache.REEF.IMRU/OnREEF/ResultHandler/WriteResultHandler.cs

```diff
@@ -83,7 +82,7 @@ public void Dispose()
 
             if (_fileSystem.Exists(remoteUri))
             {
-                Exceptions.Throw(
+                Utilities.Diagnostics.Exceptions.Throw(
```

Comment:
I added Exception in this project, it is ambiguous with the Exception in the utility project. I have to add name space to avoid conflict otherwise it causes build error.